sdks/python/apache_beam/runners/portability/fn_api_runner.py

```diff
@@ -849,6 +850,52 @@ def fuse(producer, consumer):
         stage.deduplicate_read()
       return final_stages
 
+    def inject_timer_pcollections(stages):
+      for stage in stages:
+        for transform in list(stage.transforms):
```

Comment:
Why do we need this explicit cast?  (I assume this is a set?)

Reply:
We are mutating it below.